### ISSUE TITLE
[DeepLearning] Homework 3

### ISSUE THREAD

**Issue by eunbin777** [eunbin777/MachineLearning#9]
3.1 Basic RNN model
- Generate two time-series prediction models for any time-series data with keras API
- with your own RNN model (type of recurrent layers, return sequences, …)
- Print train/validation loss of your own model
